plugins/woocommerce/includes/class-wc-privacy-erasers.php

```diff
@@ -134,7 +134,7 @@ public static function order_data_eraser( $email_address, $page ) {
 
 		$orders = wc_get_orders( $order_query );
 
-		if ( 0 < count( $orders ) ) {
+		if ( is_array( $orders ) && 0 < count( $orders ) ) {
```

Comment:
This change can be left out.  There is no reason that $orders wouldn't be an array unless `paginate` is passed into `wc_get_orders()`.  Since the code controls those parameters, it shouldn't happen.

Reply:
done https://github.com/woocommerce/woocommerce/pull/41482/commits/7cf06330d21e8267beec16506d0304b1431947f8